bookshelf/google-checks.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
```

Comment:
I'm happy to see this, but shouldn't it go in the main dir and be referenced by all the children?  @shun-fan @tswast